README.rst

```diff
@@ -13,11 +13,41 @@ Nighres
 Nighres is a Python package for processing of high-resolution neuroimaging data.
 It developed out of `CBS High-Res Brain Processing Tools
 <https://www.cbs.mpg.de/institute/software/cbs-tools>`_ and aims to make those
-tools easier to install, use and extend.
+tools easier to install, use and extend. Nighres now includes new functions from
+the `IMCN imaging toolkit <https://github.com/IMCN-UvA/imcn-imaging>`_ and 
+additional packages may be added in future releases.
 
-Because parts of the package have to be built locally it is currently not possible to use ``pip install`` directly from PyPI. Instead, please follow the installation instructions provided at http://nighres.readthedocs.io/en/latest/installation.html
+Because parts of the package have to be built locally it is currently not possible to 
+use ``pip install`` directly from PyPI. Instead, please follow the installation 
+instructions provided at http://nighres.readthedocs.io/en/latest/installation.html
 
-Currently, Nighres is developed and tested on Linux Ubuntu Trusty and Python 2.7. We hope to support all common Unix platforms and Python 3 soon.
+Currently, Nighres is developed and tested on Linux Ubuntu Trusty and Python 3.5. 
+Release versions have been extensively tested on our example data. Development 
+versions include additional modules and functions but with limited guarantees.
+
+
+Required packages
+=================
+
+In order to run Nighres, you will need:
+
+* python >= 3.5
+* Java JDK 1.7, 1.8 or 1.9
+* JCC >=3.0
+* nibabel
+* nipype
+* numpy
+* pandas
+* psutil
+
+For instance, in Debian/Ubuntu (amd64 systems):
+
+    sudo apt-get install openjdk-8-jdk
+    export JCC_JDK=/usr/lib/jvm/java-8-openjdk-amd64
+    pip3 install jcc 
+    pip3 install nibabel nipype numpy pandas psutil
```

Comment:
No, we are not Python 2 compatible (and I don't want to, to be honest, because it's that much more to check, and it gets really bad if people would try to use both at once).
pip is useless because it referesto python 2 or 3 depending on your installation. I used pip3 to make it work on my typical installations, but 'python3 -m pip' would be a better model.
I am fine with having all these things run from setup.py rather than build.sh, but you need to set the JCC_JDK variable before you install jcc